src/framework/theme/components/radio/radio-group.component.ts

```diff
@@ -182,13 +182,11 @@ export class NbRadioGroupComponent implements AfterContentInit, OnDestroy, Contr
 
   writeValue(value: any): void {
     this.value = value;
-
-    if (typeof value !== 'undefined') {
-      this.updateValues();
-    }
+    this.updateValues();
```

Comment:
You can remove this line as the `value` setter already has a `updateValues` call.